test/format/wts.c

```diff
@@ -336,6 +336,18 @@ configure_chunkcache(char **p, size_t max)
     }
 }
 
+/*
+ * configure_prefetch --
+ *     Configure prefetch settings for opening a connection. When enabled, this allows sessions to
+ *     use the prefetch feature.
+ */
+static void
+configure_prefetch(char **p, size_t max)
+{
+    if (GV(PREFETCH))
```

Comment:
As in another PR, we throw an error whenever prefetch and in-memory configuration is enabled. How about preventing it here to enable both `prefetch` and `in-memory`.

Reply:
Yes it is taken care of [here](https://github.com/wiredtiger/wiredtiger/pull/10590/files#diff-697c4698297db509ea7fd0ead993bc92c12a9e68cb1b6497a2385ac06087ee54R1105-R1106).